AntonioLok/online-shopping-reactJS
# add product lists component

Procrastinated a little bit due to holidays but I am back to working on my side project.
I followed a [tutorial](https://appdividend.com/2018/06/15/react-redux-axios-tutorial-example/?fbclid=IwAR1YweXGoPdq_Od9p0js9Q1_izwFfO9qCoqP-P36NrIfGY5KAZX7Rw7dIP8) on how to set up my redux store, I hope I did not follow a bad tutorial.

# Changelog
- Finish setting up the redux store
- Create actions and reducers for fetching products with an API call
- Add the products component. User should now see the list of all products for the selected section

# Test
- Install dependencies using `npm i`
- Start the web app using `npm run start-webapp`
- Start the web server using `npm run start-server`
- Click on men > jeans or navigate [here](http://localhost:8080/Men/Jeans)

Please note that I have only added products for the men/jeans section so if you click on others sections, it will not render any products.
